Client/RecommendationClient.php

```diff
@@ -22,4 +22,13 @@ public function updateContent($contentId);
      * @return void
      */
     public function deleteContent($contentId);
+
+    /**
+     * @param int $userId YooChoose user ID
+     * @param int $scenarioId YooChoose scenario ID
```

Comment:
I don't disagree with the principle (it can be discussed with the rest of the team), but there are a few issues with this.

First, as said above, it does need to be maintained. We have hundreds of occurences of userId. In addition, the API itself doesn't specify any format for it. Why would we limit things with an example that matches one implementation that doesn't specify an example ?

Furthermore, this interface is generic. It's not about YooChoose (namespace + class name). The YooChoose Notifier is an implementation of this Interface for the YooChoose API. I'd rather keep this separation. Therefore we can't really provide an example, as we don't know what the target system will be (yes, this has implications regarding the method's arguments themselves, but I did add a comment for it.